node/src/test/java/io/airlift/node/TestNodeConfig.java

```diff
@@ -94,11 +94,11 @@ public void testValidations()
                 .setEnvironment("test")
                 .setNodeId(UUID.randomUUID().toString()));
 
-        assertFailsValidation(new NodeConfig().setNodeId("abc/123"), "nodeId", "is malformed", Pattern.class);
+        assertFailsValidation(new NodeConfig().setNodeId("abc/123"), "nodeId", NodeConfig.ID_REGEXP_ERROR, Pattern.class);
```

Comment:
Let's put the literal string here, so we can tell that the error message is what we expect.

Reply:
Good point. Updated.